backend/habitat/viewsets.py

```diff
@@ -1099,57 +1286,57 @@ def habitat_observations(request):
     province             = params.get('province')
     ecoregion            = params.get('ecoregion')
 
-    global_archive = []
-    sediment = []
-    squidle = []
+    global_archive = None
+    sediment = None
+    squidle = None
 
     with connections['transects'].cursor() as cursor:
         try:
-            # Global Archives observations
+            # Global Archives stats
             if boundary_type == 'amp':
-                cursor.execute(SQL_GET_AMP_HABITAT_OBS_GLOBALARCHIVE, [network, park, zone, zone_iucn])
+                cursor.execute(SQL_GET_AMP_HABITAT_OBS_GLOBALARCHIVE + SQL_GET_GLOBALARCHIVE_STATS, [network, park, zone, zone_iucn])
```

Comment:
It feels vaguely dirty just munging sql strings like this, particularly when there's a shared data structure involved, and yet... it also seems like a nice efficient way to go!

Reply:
Yeah, it'd be great if I could somehow signify some query constants as fragments but 🤷